src/components/Popup/TaskQuestionnaire.js

```diff
@@ -0,0 +1,284 @@
+export default {
+    "resourceType": "Questionnaire",
+    "id": "Task",
+    "name": "Task",
+    "meta": {
+      "profile": [
+        "http://hl7.org/fhir/StructureDefinition/cqf-questionnaire",
+        "http://hl7.org/fhir/us/davinci-dtr/StructureDefinition/dtr-questionnaire-r4"
+      ]
+    },
+    "title": "Task",
+    "status": "draft",
+    "subjectType": ["Patient"],
+    "date": "2019-12-06",
+    "publisher": "Da Vinci DTR",
+    "extension": [
+      {
+        "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+        "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/HomeOxygenTherapy-prepopulation"
+      },
+      {
+        "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+        "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicPatientInfo-prepopulation"
+      },
+      {
+        "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+        "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicPractitionerInfo-prepopulation"
+      }
+    ],
+    "item": [
+      {
+        "linkId": "1",
+        "text": "Basic Info",
+        "type": "group",
+        "item": [
+            {
+                "linkId": "1.1",
+                "text": "basedOn",
+                "type": "reference",
+                "required": false
+            },
+            {
+                "linkId": "1.2",
+                "text": "status",
+                "type": "choice",
+                "required": true,
+                "answerOption": [
```

Comment:
The drop downs for these and other "choice" type questions here do not work in the UI. The only way to know a valid code to type in is to look at the source.

![image](https://user-images.githubusercontent.com/11613067/91494457-dc72e380-e886-11ea-80f4-e1722a408e81.png)

Reply:
strange, thought I had fixed that bug.